utils/generate_qr_codes.py

```diff
@@ -3,22 +3,25 @@
 import base64
 import os
 import qrcode
-import PIL as pil
+# import PIL as pil
+import io
 
 def readRandomTokens(filename):
     tokens = []
     with open(filename) as fp:
         tokens = [t.strip() for t in fp.readlines()]
     return tokens
 
-def saveAsQRCode(outdir, token):
-    qrcode_data = "nrelopenpath://login_token?token="+token
-    qrcode_img = qrcode.make(qrcode_data)
-    draw = pil.ImageDraw.Draw(qrcode_img)
-    draw.text((55,10), token, fill=0, align="center", anchor="mm")
```

Comment:
this text version of the opcode is missing from the newly rewritten value. We should retain this functionality so that admins can see which QR code they are handing out without having to use a QR code scanner, which may not display the full opcode anyway if it is long. Note that this may require re-adding PIL